elastic/kibana
# [SecuritySolution] add global filter to topN

## Summary

https://github.com/elastic/kibana/issues/90418

TopN chart in alerts table didn't take alerts status and ruled id as part of its filter, and caused the numbers in topN chart does not match with alerts table.
We don't show numbers in alerts' topN chart anymore, but we can see the number is matched between alerts table and query response.

Steps to verify:
1. Create a rule
2. Go to rule's details page
3. Hover on to a cell in alerts table and click view `topN chart` button.
4. Click the inspect icon on the chart panel, click on Request tab and inspect the query. The filter should contain rule status and rule id.
5. Click on the Response tab, `hits.total` should equal to the number of alerts appears on alerts table.

<img width="2541" alt="Screenshot 2021-09-16 at 11 43 04" src="https://user-images.githubusercontent.com/6295984/133601105-fa4a732e-4e97-4ecd-8a6e-ea4878658d18.png">

<img width="2557" alt="Screenshot 2021-09-16 at 12 18 17" src="https://user-images.githubusercontent.com/6295984/133603085-443d749d-acea-41eb-97e8-86418950fac6.png">
